backend/lib/database/postgres/methods.py

```diff
@@ -55,18 +55,17 @@ def _get_data(conn, sql_class, columns_to_select, column_to_value_map):
     return {}
 
 
-def _safe_core_insert(conn, sql_class, row):
+def _safe_core_insert(conn, sql_class, row, unique_column):
```

Comment:
Please update the docstring to say what these things are.

Reply:
Should this be a list of unique columns?

Reply:
One unique is enough it is just to return an ID